hotel_roommatik/models/inherited_hotel_folio.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 Jose Luis Algara (Alda hotels) <osotranquilo@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import json
+
+from odoo import api, models
+
+
+class HotelFolio(models.Model):
+
+    _inherit = 'hotel.folio'
+
+    @api.model
+    def rm_get_reservation(self, Code):
+        folio_res = self.env['hotel.folio'].search([('id', '=', Code)])
```

Comment:
Here can use 'browse' instead of 'search'.
- 'browse' method uses prefetching (cache)
- 'search' method launch a sql query (it's a more complex method)

Reply:
We are waiting to define the search fields , but they are more than id (id, name, channelid, etc... ), what's more, when the id is defined it probably does not make sense.